Sources/Credentials/MultiTypeSafeCredentials.swift

```diff
@@ -72,7 +72,7 @@ public protocol TypeSafeMultiCredentials: TypeSafeCredentials {
 
 extension TypeSafeMultiCredentials {
     
-    /// Static function that attempts to create an instance of Self by iterating through an array `TypeSafeCredentials` types and calling `authenticate`. On a successful authentication, an instance of Self is initialized from the `TypeSafeCredentials` instance and returned so it can be used by a `TypeSafeMiddleware` route. On a failed authentication, an unauthorized response is sent immediately. If the authentication header isn't recognised, authenticate is called on the next `TypeSafeCredentials` type.
+    /// Static function that attempts to create an instance of Self by iterating through the array `TypeSafeCredentials` types and calling authenticate. On a successful authentication, an instance of Self is initialized and returned for use in a Codable route. On a failed authentication, an unauthorized response is sent immediately. If the authentication header isn't recognised, authenticate is called on the next type in the array.
```

Comment:
Missing word: through the array **of** `TypeSafeCredentials` types

I also think we should clarify "failed authentication" and "header isn't recognised": I think people unfamiliar with the pass/fail/skip mechanics would find this explanation a little confusing.  Maybe we can describe the process in a clearer way?